### ISSUE TITLE
duplicated client type hint in vscode

### ISSUE THREAD

**Issue by lisonge** [lisonge/vite-plugin-monkey#44]
```ts
// vite-env.d.ts
/// <reference types="vite/client" />
declare module '$' {
  export * from 'vite-plugin-monkey/dist/client';
}
```

![image](https://user-images.githubusercontent.com/38517192/218249486-204c8ae5-e1dd-45b6-bd4c-8f6e88bc05c3.png)

**Comment by lisonge** [comment]
`$` and `vite-plugin-monkey/dist/client` are duplicated

**Comment by lisonge** [comment]
# The Solution

##  when use  module `$`
```ts
// vite-env.d.ts
/// <reference types="vite/client" />
/// <reference types="vite-plugin-monkey/client" />
```
![image](https://user-images.githubusercontent.com/38517192/218249878-9346259a-e576-4081-be09-e1872252750b.png)

##  when use `global variables`

if you use `mountGmApi=true`, you can use `vite-plugin-monkey/global`
```ts
// vite-env.d.ts
/// <reference types="vite/client" />
/// <reference types="vite-plugin-monkey/global" />
```
![image](https://user-images.githubusercontent.com/38517192/218249874-cdca6e3f-3351-404c-a9fe-f02370c374ff.png)

**Comment by lisonge** [comment]
fixed by [v2.12.0](https://github.com/lisonge/vite-plugin-monkey/blob/v2.12.0/packages/vite-plugin-monkey/CHANGELOG.md)
